feature-libs/cart/base/components/added-to-cart-dialog/added-to-cart-dialog.component.ts

```diff
@@ -31,6 +34,13 @@ export class AddedToCartDialogComponent {
   addedEntryWasMerged$: Observable<boolean>;
   promotionLocation: PromotionLocation = PromotionLocation.ActiveCart;
 
+  focusConfig: FocusConfig = {
+    trap: true,
+    block: true,
+    autofocus: "button",
+    focusOnEscape: true,
+  };
```

Comment:
I have been testing this component with this change, using the focus config, and noticed a change to a desired existing behaviour. If you look in `added-to-cart-dialog.component.html` line 83, the `<a>` has `autofocus` property so that each time the modal is opened the focus shifts to that element. Now using this config the focus is lost after closing with esc key. 

The way to see what I mean is click on 'Add to Cart' for a product. Then hit esc key, and click 'Add to Cart' again. Notice that the 'View Cart' is not focused on second pop up (also esc key is not handled on 2nd attempt)

Reply:
This comment addresses this: https://github.com/SAP/spartacus/pull/15390/files#r828395795